graspologic/align/seeded_procrustes.py

```diff
@@ -0,0 +1,186 @@
+
+    def fit(
+        self,
+        X,
+        Y,
+        seeds,
+        verbose=False,
+    ) -> "SeededProcrustes":
+
+        """
+        Overrides the fit method from the parent class ``BaseAlign`` to find
+        a value for ``Q`` given two datasets, ``X`` and ``Y``, and a matrix of seeds.
+
+        Parameters
+        ----------
+        X : np.ndarray, shape (n, d)
+            Dataset to be mapped to ``Y``, must have same number of dimensions
+            (axis 1) as ``Y``.
+
+        Y : np.ndarray, shape (m, d)
+            Target dataset, must have same number of dimensions (axis 1) as ``X``.
+
+        seeds : np.ndarray, shape (?,2)
+            Matrix of pairs, demonstrates the relation between rows of ``X`` and ``Y``.
+            The first column pertains to the ``X`` component of the pairs, and the second column
+            pertains to the ``Y`` component. The pairs are stored in the rows of ``seeds``,
+            each row containing an ``X`` value and a corresponding ``Y`` value.
+
+        verbose : bool
+            Degree to which the code records and produces feedback when run.
+
+        Returns
+        -------
+        self : returns an instance of self
+        """
+
+        n = len(X[0])
+        init_Q = _find_Q_from_pairs(X, Y, seeds)
+        procruster = SeedlessProcrustes(
+            init="custom",
+            initial_Q=init_Q,
+            optimal_transport_eps=1.0,
+            optimal_transport_num_reps=100,
+            iterative_num_reps=10,
+        )
+        currtime = time.time()
+        X_mapped = procruster.fit_transform(X, Y)
+        self.Q_ = procruster.Q_
+
+        if verbose > 1:
+            print(
+                f"{time.time() - currtime:.3f} seconds elapsed for SeedlessProcrustes."
+            )
+        X = (X_mapped[:n], X_mapped[n:])
+        return self
+
+
+def _find_Q_from_pairs(X: np.ndarray, Y: np.ndarray, seeds: np.ndarray) -> np.ndarray:
+    """
+    This function uses the seeds matrix to sort the matrices X and Y
+    by their paired indices.
+
+    Parameters
+    ----------
+    X : np.ndarray, shape (n, d)
+        Dataset to be mapped to ``Y``, must have same number of dimensions
+        (axis 1) as ``Y``.
+
+    Y : np.ndarray, shape (m, d)
+        Target dataset, must have same number of dimensions (axis 1) as ``X``.
+
+    seeds : np.ndarray, shape (?, 2)
+        Matrix of pairs between rows of ``X`` and ``Y``
+
+    Returns
+    -------
+    op.Q_ : a call of the ``Q_`` attribute of the OrthogonalProcrustes class
+
+    """
+    paired_inds1 = seeds[:, 0]  # first col of seeds relates to X
+    paired_inds2 = seeds[:, 1]  # second col of seeds relates to Y
+    Xp = X[paired_inds1, :]  # X is sorted in terms of the first col indexes
+    Yp = Y[paired_inds2, :]  # Y is sorted in terms of the second col indexes
+    op = OrthogonalProcrustes()  # new instance of orthogonal pro.
+    op.fit(Xp, Yp)  # call fit on instance of orthogonal pro. to get Q
```

Comment:
```suggestion
    op.fit(Xp, Yp)
```
also clear from the code